index.html

```diff
@@ -1,17 +1,240 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <!-- <script src="https://cdn.tailwindcss.com"></script> -->
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Learn Scraping By Doing</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
ain't merging bc this
![931587536592191491](https://user-images.githubusercontent.com/40650681/150594143-22112f98-e1b9-4e2c-ab0b-0542597279e1.jpg)

Reply:
Resolved